app/mutations/application/createApplicationReviewMutation.ts

```diff
@@ -12,9 +12,20 @@ const mutation = graphql`
   ) {
     createApplicationReviewMutationChain(input: $input) {
       applicationReview {
-        id
-        formResultId
-        reviewStatus
+        ...ApplicationReviewContainer_applicationReview
+        ...ApplicationCommentsByForm_applicationReview
+        formResultByFormResultId {
+          applicationReviewsByFormResultId {
```

Comment:
I see above that `reviewCommentsByApplicationReviewId` has an `@connection` declared. Are we updating one here with this edge list? Note that by refetching the list of `id` fields only we're triggering a rebuild of the edges list. If we were to add a `@connection` handler, we would be able to insert just the new edge into the list without rebuilding it.

Reply:
What's should we do?

Reply:
If this is working, let's leave as-is for now and optimize payload size in a subsequent PR